landingzones/views.py

```diff
@@ -221,6 +222,70 @@ def submit_create(
             raise ex
 
 
+class ZoneUpdateMixin(ZoneConfigPluginMixin):
+    """Mixin to be used in zone update in UI and REST API views"""
+
+    def submit_update(self, zone, request=None):
+        """
+        Handle timeline updating and taskflow initialization after a LandingZone
+        object has been updated.
+
+        :param zone: LandingZone object
+        :raise: taskflow.FlowSubmitException if taskflow submit fails
+        """
+        # taskflow = get_backend_api('taskflow')
+        timeline = get_backend_api('timeline_backend')
+        tl_event = None
+        user = request.user if request else None
+
+        # Add event in Timeline
+        if timeline:
+            tl_extra = {
+                'title': zone.title,
+                'assay': str(zone.assay.sodar_uuid),
+                'description': zone.description,
+            }
+            tl_event = timeline.add_event(
+                project=zone.project,
+                app_name=APP_NAME,
+                user=user,
+                event_name='zone_update',
+                description='update landing zone {{{}}} for {{{}}} in '
+                '{{{}}}'.format('zone', 'user', 'assay'),
+                status_type='SUBMIT',
```

Comment:
In this case we are simply updating the LandingZone object synchronously and no taskflow is used. Hence, the status should be set to `OK` here. Right now these appear as stuck in the `SUBMIT` status in timeline:

![2023-04_zone_update_tl_bug](https://user-images.githubusercontent.com/11211145/235094143-7b2ee946-4d19-40c1-8d20-e3dac4dfba7e.png)